src/autorest/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "alwaysStrict": true,
+    "forceConsistentCasingInFileNames": true,
```

Comment:
We should probably try to go to great lengths to ensure that all the tsconfig files are kept as close to exactly the same as possible, given the massive amount of reuse/sharing we're looking at doing.